sims.mk

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/make -f
```

Comment:
Why creating a dedicated makefile for simulation tests and not reusing Makefile?

Our default Makefile is already general purpose